build_run.sh

```diff
@@ -0,0 +1,10 @@
+#RUN npm install --unsafe-perm=true
```

Comment:
**build.sh** creates a local swarm and runs the visualizer localy for testing.

**push.sh** is a script that pushes to dockerhub and should be **removed** from the repo (it points to my own dockerhub repo)